dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/FATSuite.java

```diff
@@ -50,9 +50,8 @@ public class FATSuite {
 
     public static final String BETA_ID = MicroProfileActions.MP61_ID + "_BETA";
 
-    public static RepeatTests aboveMP50Repeats(String serverName) {
-        return TelemetryActions
-                        .repeat(serverName, TelemetryActions.MP50_MPTEL11, MicroProfileActions.MP60, MicroProfileActions.MP61)
-                        .andWith(FeatureReplacementAction.EE10_FEATURES().withBeta().fullFATOnly().withID(BETA_ID));
+    public static RepeatTests allMPRepeats(String serverName) {
+        return MicroProfileActions
+                        .repeat(serverName, MicroProfileActions.MP61, TelemetryActions.MP14_MPTEL11,  TelemetryActions.MP41_MPTEL11, TelemetryActions.MP50_MPTEL11, MicroProfileActions.MP60);
```

Comment:
It looks like the servlet tracing still only happens in beta mode so we still need a beta repeat.

However, we ideally need it for MP60 and MP61 because there's a separate servlet tracing implementation on each which is a little more complicated.

Reply:
You need to use `TelemetryActions.repeat` here, not `MicroProfileActions.repeat`